wpilibc/athena/src/PIDController.cpp

```diff
@@ -503,11 +503,11 @@ bool PIDController::OnTarget() const {
   double error = GetAvgError();
   switch (m_toleranceType) {
     case kPercentTolerance:
-      return std::fabs(error) <
+      return std::abs(error) <
```

Comment:
This seems unrelated to PMD.  std::fabs is a perfectly fine function to use here.